docs/developer-guide/client/README.md

```diff
@@ -0,0 +1,21 @@
+# C++ Client Documentation
+
+## Table of Contents
+- [Introduction](#introduction)
+- [Raylib Library](#raylib-library)
+
+## Introduction
+The C++ client is a simple game client that uses the Raylib library to handle graphics and audio. The client is designed to be simple and easy to use, but it is also very flexible and can be used to create more complex games.
+
+## Raylib Library
+We use the Raylib library to handle our graphics and audio. Raylib is a simple and easy-to-use library that provides a wide range of features. The Raylib library is written in C, but we have wrapped it in C++ classes to make it easier to use.
+
+Explore the following sections to understand and utilize different aspects of the Raylib library:
+
+- [**Audio**](audio/index.html): Learn how to add sounds, music, and other audio effects to your game.
```

Comment:
I didn't manage to put audio/README.md, it directed me to the wrong thing. I checked the issues on mdbook's github that talked about this but I couldn't solve the problem.